xrviz/style.py

```diff
@@ -0,0 +1,42 @@
+import panel as pn
+from holoviews.plotting import list_cmaps
+from .sigslot import SigSlot
+
+
+class Style(SigSlot):
+
+    def __init__(self):
+        super().__init__()
+        self.height = pn.widgets.IntSlider(name='height', value=300, start=100, end=1200)
+        self.width = pn.widgets.IntSlider(name='width', value=700, start=100, end=1200)
+        self.cmap = pn.widgets.Select(name='cmap', value='Inferno',
+                                      options=list_cmaps(provider='bokeh', reverse = False))
+        self.colorbar = pn.widgets.Checkbox(name='colorbar', value=True)
+        # colormap_limits
+        self.lower_limit = pn.widgets.TextInput(name='cmap lower limit', width=140)
```

Comment:
Is an absolute width necessary here? I imagine it will turn out differently for someone with a resolution very different from yours

Reply:
I think it can be useful, since all we need is a box small enough to enter 1-2 digits usually and max 6-7 is values are large or minor decimal changes are needed. So it is better to have a box of width fixed for that.

Reply:
Yes, but the point is the actual visual size will depend on the user's system. It is better to use auto-sizing whenever possible. However, I will l leave it for now.